presto-hive/src/main/java/com/facebook/presto/hive/HivePageSink.java

```diff
@@ -283,7 +284,7 @@ private HiveRecordWriter createWriter(List<Object> partitionRow)
         checkArgument(partitionRow.size() == partitionColumnNames.size(), "size of partitionRow is different from partitionColumnNames");
 
         List<String> partitionValues = partitionRow.stream()
-                .map(Object::toString) // todo this seems wrong
+                .map(Objects::toString)
```

Comment:
I believe this will set the partition key to the string `"null"`, but Hive uses a special stand in value for nulls.  Additionally someone will need to verify the other types are correctly being converted to partition key values.